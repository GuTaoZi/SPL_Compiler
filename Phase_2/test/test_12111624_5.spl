struct aa {
    int bb;
    float cc;
    char dd;
} ee, ff, gg[214748];
int www;

int main(){
    gg[114514] = ee = ff;
    ff.bb = ee.bb + www;

    gg[0].dd = 123;
}