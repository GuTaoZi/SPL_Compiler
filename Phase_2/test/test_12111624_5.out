Error type 5 at Line 12: unmatching types for assign(=).
